sdk/search/Azure.Search.Documents/src/Indexes/Models/EntityRecognitionSkill.SkillVersion.cs

```diff
@@ -25,6 +25,7 @@ public SkillVersion(string value)
             }
 
             /// <summary> <see cref="SkillVersion.V1"/> version of the <see cref="EntityRecognitionSkill"/> is deprecated. Use the  <see cref="SkillVersion.V3"/> version instead. </summary>
+            [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Do we also want to make the `EntityRecognitionSkill` constructor that defaults to this version EditorBrowsableState.Never as well? Same comment for `SentimentSkill`.